test/services/github_client_test.rb

```diff
@@ -0,0 +1,38 @@
+require 'test_helper'
+
+class GithubClientTest < ActiveSupport::TestCase
```

Comment:
I would also like to get feedback on this. VCR is really cool BTW. What do you think about the test coverage / style?